authors/apps/profiles/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+
+from .views import ProfileRetrieveAPIView, RetrieveProfilesView
+
+
+app_name = 'profiles'
+urlpatterns = [
+    path('profiles/<str:username>', ProfileRetrieveAPIView.as_view()),
```

Comment:
You have not yet added a slash to this URL on line 8.
Since we're following a convention where all our URLs have / at the end, do the same for this as well.

Reply:
sorry about that, I have fixed it